iwf/state_options.go

```diff
@@ -5,12 +5,16 @@ import (
 )
 
 type StateOptions struct {
-	SearchAttributesLoadingPolicy *iwfidl.PersistenceLoadingPolicy
-	DataAttributesLoadingPolicy   *iwfidl.PersistenceLoadingPolicy
-	WaitUntilApiTimeoutSeconds    *int32
-	ExecuteApiTimeoutSeconds      *int32
-	WaitUntilApiRetryPolicy       *iwfidl.RetryPolicy
-	ExecuteApiRetryPolicy         *iwfidl.RetryPolicy
-	WaitUntilApiFailurePolicy     *iwfidl.WaitUntilApiFailurePolicy
-	ExecuteApiFailureProceedState WorkflowState
+	WaitUntilApiSearchAttributesLoadingPolicy *iwfidl.PersistenceLoadingPolicy
+	ExecuteApiSearchAttributesLoadingPolicy   *iwfidl.PersistenceLoadingPolicy
+	SearchAttributesLoadingPolicy             *iwfidl.PersistenceLoadingPolicy
+	WaitUntilApiDataAttributesLoadingPolicy   *iwfidl.PersistenceLoadingPolicy
+	ExecuteApiDataAttributesLoadingPolicy     *iwfidl.PersistenceLoadingPolicy
```

Comment:
Similar to https://github.com/indeedeng/iwf-python-sdk/pull/37/files#r1805101514 we can sort them and add some comments to make it more friendly

Reply:
Thanks! Done now ✅